pkg/access/commands/use.go

```diff
@@ -1,60 +1,68 @@
 package commands
 
 import (
+	"context"
 	"encoding/json"
-	"errors"
 	"fmt"
+	"os/exec"
+	"strings"
 
 	"github.com/spf13/cobra"
 
 	"github.com/apono-io/apono-cli/pkg/aponoapi"
+	"github.com/apono-io/apono-cli/pkg/clientapi"
 )
 
-const outputFlagName = "output"
+const (
+	outputFlagName = "output"
+	runFlagName    = "run"
+)
 
 func AccessDetails() *cobra.Command {
 	var outputFormat string
+	var run bool
 
 	cmd := &cobra.Command{
 		Use:     "use [id]",
 		GroupID: Group.ID,
 		Short:   "Get access session details",
-		Args:    cobra.ExactArgs(1), // This will enforce that exactly 1 argument must be provided
+		Args:    cobra.MinimumNArgs(1), // This will enforce that exactly 1 argument must be provided
 		RunE: func(cmd *cobra.Command, args []string) error {
 			client, err := aponoapi.GetClient(cmd.Context())
 			if err != nil {
 				return err
 			}
 
-			id := args[0]
-			accessDetails, _, err := client.ClientAPI.AccessSessionsAPI.GetAccessSessionAccessDetails(cmd.Context(), id).Execute()
+			accessID := args[0]
+
+			accessDetails, _, err := client.ClientAPI.AccessSessionsAPI.GetAccessSessionAccessDetails(cmd.Context(), accessID).Execute()
+			if err != nil {
+				return err
+			}
+
+			if run {
+				return executeAccessDetails(cmd.Context(), accessID, accessDetails)
+			}
+
+			err = verifyOutputFormatIsSupported(cmd.Context(), client, accessID, outputFormat)
 			if err != nil {
 				return err
 			}
 
 			var output string
 			switch outputFormat {
 			case "cli":
```

Comment:
I'm missing something, where is the "no output format" option (default)?

Reply:
default is cli
```go
flags.StringVarP(&outputFormat, outputFlagName, "o", "cli", "output format")
```